pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -70,30 +73,21 @@ func (o *objectWatcherImpl) Create(clusterName string, desireObj *unstructured.U
 		return err
 	}
 
-	fedKey, err := keys.FederatedKeyFunc(clusterName, desireObj)
-	if err != nil {
-		klog.Errorf("Failed to get FederatedKey %s, error: %v", desireObj.GetName(), err)
-		return err
-	}
-	_, err = helper.GetObjectFromCache(o.RESTMapper, o.InformerManager, fedKey)
+	clusterObj, err := dynamicClusterClient.DynamicClientSet.Resource(gvr).Namespace(desireObj.GetNamespace()).Create(context.TODO(), desireObj, metav1.CreateOptions{})
 	if err != nil {
-		if !apierrors.IsNotFound(err) {
-			klog.Errorf("Failed to get resource %v from member cluster, err is %v ", desireObj.GetName(), err)
-			return err
+		if apierrors.IsAlreadyExists(err) {
+			klog.Infof("Skip creating resource(kind=%s, %s/%s) in cluster %s because it already exists", desireObj.GetKind(), desireObj.GetNamespace(), desireObj.GetName(), clusterName)
+			return nil
```

Comment:
I believe that an error should still be returned here. 
The situation where the resource already exists should not be ignored.

Reply:
Done.